input/pagecontent/medicine-information.md

```diff
@@ -14,7 +14,7 @@ It is anticipated that future releases of AU Core will define AU Core profiles o
 - MedicationDispense (with AU Core Medication) are used to support dispense records and medication management use cases.
 - MedicationStatement (with AU Core Medication) are used to support summary statements of medicine use.
 
```

Comment:
Add a sentence here: The guidance below addresses how medicinal product identification can be structured in FHIR conformant to AU Core.

Reply:
Added, thanks.